apps/pie-microsite/package.json

```diff
@@ -1,22 +1,24 @@
 {
   "name": "pie-microsite",
   "description": "todo",
-  "version": "1.17.0",
+  "version": "1.18.0",
   "main": "index.js",
   "keywords": [],
   "author": "JustEatTakeaway - Vue Design System Team",
   "license": "MIT",
-  "packageManager": "yarn@3.2.1",
   "scripts": {
-    "watch:sass": "sass --watch --load-path=../../node_modules/ src/assets/styles:dist/assets/styles",
-    "watch:eleventy": "eleventy --serve",
+    "build": "npm-run-all build:eleventy build:sass post-css",
     "build:sass": "sass --load-path=../../node_modules/ src/assets/styles:dist/assets/styles",
     "build:eleventy": "eleventy",
-    "dev": "npm-run-all build:sass --parallel watch:sass watch:eleventy post-css",
-    "build": "npm-run-all build:eleventy build:sass post-css",
+    "clean:sass": "rimraf dist/assets/styles",
```

Comment:
doesn't rm -rf work here instead?

Reply:
rimraf works universally across Unix and Windows OS. `rm -rf` will fail on Windows machines